src/NovelRT/Timing/StepTimer.cpp

```diff
@@ -7,9 +7,9 @@
 
 namespace NovelRT::Timing {
   StepTimer::StepTimer(uint32_t targetFrameRate, double maxSecondDelta) :
-    _frequency(SDL_GetPerformanceFrequency()),
+    _frequency(glfwGetTimerFrequency()),
```

Comment:
This fails and defaults to 0 as GLFW is not initialized yet - refer to the initialization order in `NovelRunner.cpp`.